CribblyBackend/Controllers/GameController.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using CribblyBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CribblyBackend.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class GameController : ControllerBase
+    {
+        private readonly IGameService GameService;
+        public GameController(IGameService GameService)
+        {
+            this.GameService = GameService;
+        }
+
+        /// <summary>
+        /// GetById fetches the specified Game.
+        /// </summary>
+        /// <param name="id">The id of the Game to get</param>
+        /// <returns></returns>
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetById(int id)
+        {
+            var p = await GameService.GetById(id);
+            if (p != null)
+            {
+                return Ok(p);
+            }
+            return NotFound();
+        }
+        
+        /// <summary>
+        /// Create makes a new Game object.
+        /// </summary>
+        /// <param name="Game">The Game object that will be created</param>
+        /// <returns></returns>
+        [HttpPost]
+        public async Task<IActionResult> Create([FromBody] Game Game)
```

Comment:
🌵 📖 same as the comment above, it can be helpful for params to be camelCase 🤷‍♂️